Modules/socketmodule.c

```diff
@@ -5611,6 +5611,20 @@ static PyType_Spec sock_spec = {
     .slots = sock_slots,
 };
 
+static PyObject*
+socket_getattr(PyObject *self, PyObject *name)
+{
+    if (PyUnicode_EqualToUTF8(name, "SocketType")) {
+        socket_state *state = get_module_state(self);
+        PyErr_Warn(PyExc_DeprecationWarning, "_socket.SocketType is deprecated and "
+                                             "will be removed in Python 3.16. "
+                                             "Use socket.socket instead");
+        return state != NULL ? (PyObject *)state->sock_type : NULL;
```

Comment:
Should `sock_type` get `INCREF`'d? I'm assuming its a static type and therefore immortal, but we should add a comment clarifying that.

Reply:
I tried to search for situations where `sock_type` is used in `socketmodule.c`. In the scenario where `Py_DECREF` is used, only `socket_clear` uses `Py_DECREF`. And `sock_type ` does not use `Py_INCREF`. And in other parts, it seems that there is no special comment.